src/components/project/dto/list-projects.dto.ts

```diff
@@ -78,6 +78,22 @@ export abstract class ProjectFilters {
   @ValidateNested()
   readonly modifiedAt?: DateTimeFilter;
 
+  @Field({
+    nullable: true,
+    description: 'Only projects with start dates within this time range',
```

Comment:
"within this time range" - not really accurate. I'd be fine with just removing. The description doesn't really add any color beyond what the fields & types communicate.